packages/commonwealth/client/scripts/models/Group.ts

```diff
@@ -24,6 +24,7 @@ interface APIResponseFormat {
       token_id?: string;
     };
   }[];
+  GroupPermissions: { Topic: unknown }[];
```

Comment:
this doesn't actually get returned from `/groups`, why did we add this?

<img width="440" alt="image" src="https://github.com/user-attachments/assets/dac6266b-9eb0-4804-afcb-73cd32eb699c">

Reply:
Good catch. Only included if some GroupPermissions exist for the group and includeTopics=true, otherwise returns undefined. I will update the model to make it optional.